docs/DESIGN.md

```diff
@@ -138,6 +138,12 @@ Two issues can arise from incorrect versioning:
 1. Version required by the OpenTelemetry .NET SDK or the instrumentations not being met.
 2. Multiple versions of the assembly in the same process since these are treated as totally independent by the runtime.
 
+#### Configuration Resolution
+
+.NET Core [Framework-dependent deployment](https://docs.microsoft.com/en-us/dotnet/core/deploying/deploy-with-cli#framework-dependent-deployment) apps could use
+[DOTNET_ADDITIONAL_DEPS](https://github.com/dotnet/runtime/blob/main/docs/design/features/additional-deps.md) and
+[DOTNET_SHARED_STORE](https://docs.microsoft.com/en-us/dotnet/core/deploying/runtime-store) from tracer-home directory to resolve these assembly conflicts.
```

Comment:
I am not sure about `tracer-home` naming. I find it cryptic. In one of the previous PRs I described it as `Installation location.`.